docs/infra/set-up-aws-account.md

```diff
@@ -7,7 +7,8 @@ The AWS account setup process will:
 
 ## Prerequisites
 
-First, [set up infrastructure tools](./set-up-infrastructure-tools.md), like Terraform, AWS CLI, and AWS authentication.
+First, [set up infrastructure tools](./set-up-infrastructure-tools.md), like Terraform, AWS CLI, and AWS authentication.     
+Second, [Set up infrastructure as code (IaC) management resources](/template-only-docs/set-up-infrastructure-as-code.md) i.e. project name, project owner, repository URL and default AWS region.
```

Comment:
Reflecting on this a bit, I feel like the page "[set up infrastructure as code (IaC) management resources](https://github.com/navapbc/template-infra/blob/284-help-new-projects-remember-to-set-up-project-config-first/template-only-docs/set-up-infrastructure-as-code.md)" is kind of unnecessary. 

What do you think about instead replacing [this instruction in the top level README](https://github.com/navapbc/template-infra/blob/284-help-new-projects-remember-to-set-up-project-config-first/README.md?plain=1#L33)

> 2. Set up resources needed to manage infrastructure as code (IaC)

with something like

> 2. Set up project
> 3. Set up AWS account(s)

Reply:
went ahead and tried this out in https://github.com/navapbc/template-infra/pull/288 lemme know if you thnk that's cleaner

Reply:
It makes sense